deploy-0chain/action.yml

```diff
@@ -67,6 +67,18 @@ inputs:
   TENDERLY_FORK_ID:
     required: true
     description: 'Tenderly fork ID is used to use Tenderly fork simulations for bridge tests'
+  zs3_client:
```

Comment:
There are a limited number of actions params, is it possible to assume if zs3 client is staging, then search api and minio will also be staging? if so can we combine these to a single param?